modules/platform/nuxeo-platform-video/nuxeo-platform-video-core/src/test/java/org/nuxeo/ecm/platform/video/extension/TestVideoImporterAndListeners.java

```diff
@@ -452,9 +449,46 @@ public void testConfiguration() throws Exception {
         // check storyboard
         @SuppressWarnings("unchecked")
         List<Map<String, Serializable>> storyboard = (List<Map<String, Serializable>>) docModel.getPropertyValue(
-                "vid:storyboard");
+                VideoConstants.STORYBOARD_PROPERTY);
         assertNotNull(storyboard);
         assertEquals(2, storyboard.size());
     }
 
+    @Test
+    @Deploy("org.nuxeo.ecm.platform.video.core:test-video-conversions-enabled.xml")
+    @Deploy("org.nuxeo.ecm.platform.video.core:test-video-listeners-contrib.xml")
+    @Deploy("org.nuxeo.ecm.platform.video.core:video-configuration-override.xml")
+    @SuppressWarnings("unchecked")
+    public void testNotConversionWhenDisabled() throws IOException {
```

Comment:
```suggestion
    public void testNoConversionWhenDisabled() throws IOException {
```

Reply:
already done before I see you review :) thx